src/ann.py

```diff
@@ -0,0 +1,50 @@
+import pandas
+from sklearn import preprocessing
+from sklearn.metrics import classification_report, confusion_matrix
+from sklearn.metrics import plot_confusion_matrix
+from sklearn.model_selection import train_test_split
+from sklearn.neural_network import MLPClassifier
+from sklearn.preprocessing import StandardScaler
+from sklearn.svm import SVC
+
+
+class ArtificialNeuralNetwork:
+    @staticmethod
+    def data_sweep():
+        # Assigning column names to the dataset
+        names = ["itching", "skin_rash", "chills", "vomiting", "fatigue", "prognosis"]
+        # reading data to panda dataframe
+        data = pandas.read_csv("../csv/Example_ANN_Training_Samples.csv", names=names)
+        # assign data from first five columns to X variable
+        x = data.iloc[:, 0:5]
+        # assign data from first fifth columns to y variable
+        y = data.select_dtypes(include=[object])
+        le = preprocessing.LabelEncoder()
+        y = y.apply(le.fit_transform)
+        return x, y
+
+    @staticmethod
+    def ann_train_test_split(x, y):
+        sweep_x = x
+        sweep_y = y
+        x_train, x_test, y_train, y_test = train_test_split(
+            sweep_x, sweep_y, test_size=0.20
+        )
+        # Feature Scaling for uniform evaluation
+        scaler = StandardScaler()
+        scaler.fit(x_train)
+        x_train = scaler.transform(x_train)
+        x_test = scaler.transform(x_test)
+        return x_train, x_test, y_train, y_test

```

Comment:
I wasn't sure how you intended to use this, so I adjusted the returns the way I thought you intended to use them. @alwinbaby does this look like what you wanted?

Reply:
As long as other def functions can use those its good